assets/css/detours/_detour_map.scss

```diff
@@ -1,5 +1,15 @@
 @use "../color/definitions" as semantic;
 
+.c-detour_map--map__clickable {
+  cursor: url("data:image/svg+xml,%3Csvg width='14' height='14' viewBox='0 0 14 14' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cstyle%3E .color %7B fill: %236D39AC; %7D %3C/style%3E%3Ccircle cx='7' cy='7' r='6' class='color' style='filter: blur(0.5px)'/%3E%3Ccircle cx='7' cy='7' r='4.5' class='color' stroke='white'/%3E%3C/svg%3E")
+      11 11,
+    pointer;
+
+  .c-vehicle-map {
+    cursor: unset;
+  }
```

Comment:
question: what is un-setting the cursor doing here? It seems like it'd overwrite the cursor property on ln4

Reply:
Ah! I was mistaking `unset` for `revert`, thanks for explaining!!